uiowa/uiowa
# [INRC] Nonprofit Organization

Resolves https://github.com/uiowa/uiowa/issues/7900

# How to test

```
ddev blt ds --site=inrc.law.uiowa.edu && ddev drush @lawinrc.local uli /node/add/nonprofit_org
```
- Add a term at `/admin/structure/taxonomy/manage/nonprofit_organization_category/add`
- Compare fields to those listed at https://github.com/uiowa/uiowa/issues/7900
- Add a nonprofit org (https://inrc.law.uiowa.edu/non-profit-organizations/abbe-center-community-mental-health)
- Compare to see that this uses content type best practices as described at https://docs.prod.drupal.uiowa.edu/development-workflow/content-type-best-practices.
